### PR TITLE
updated AAD to Entra Id in BRS

### PR DESCRIPTION
**Pull request by rajatgoyalMS**
Changed the AAD to Entra Id in BRS files for following 2 controls - 
1. Azure_KubernetesService_AuthN_Enabled_AAD
2. Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN

### CODE REVIEW

File: Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1 [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ﻿<###
 # Overview:
-    This script is used to delete all non-Azure Active Directory (AAD) identity providers from all API Management services in a Subscription.
+    This script is used to delete all non-Azure Active Directory (Entra Id) identity providers from all API Management services in a Subscription.
```
Comment (aakashpahwa-msft) [inline Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1:3]: AAD should be in brackets, and be consistent with how you're using Entra ID,
1. Microsoft Entra ID
2. Microsott Entra Id
3. Entra ID
4. Entra Id

Need to pick one

File: Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1 [hunk 2]
```diff
@@ -193,7 +193,7 @@ function Delete-NonAadIdentityProvidersInApiManagementServices
     $nonApplicableSkus = @("Consumption")
 
     Write-Host $([Constants]::DoubleDashLine)
-    Write-Host "[Step 1 of 4] Prepare to delete all non-AAD identity providers (and basic authentication) from the API Management services in Subscription: [$($SubscriptionId)]"
+    Write-Host "[Step 1 of 4] Prepare to delete all non-AAD(Entra Id) identity providers (and basic authentication) from the API Management services in Subscription: [$($SubscriptionId)]"
```
Comment (aakashpahwa-msft) [inline Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1:196]: brackets need spaces before it
This gets printed out on our customer's consoles, we can't be shabby

File: Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1 [hunk 3]
```diff
@@ -743,13 +743,13 @@ class SetAADProfile
...
 
-            # API to set AAD Profile config to Kubernetes cluster
+            # API to set Entra Id Profile config to Kubernetes cluster
             $response = Invoke-WebRequest -Method $method -Uri $armUri -Headers $headers -Body $jsonString -UseBasicParsing
             $content = ConvertFrom-Json $response.Content
         }
         catch
         {
-            Write-Host "Error occurred while setting AAD profile to Kubernetes Cluster. ErrorMessage [$($_)]" -ForegroundColor $([Constants]::MessageType.Error)
+            Write-Host "Error occurred while setting Entra Id profile to Kubernetes Cluster. ErrorMessage [$($_)]" -ForegroundColor $([Constants]::MessageType.Error)
```
Comment (aakashpahwa-msft) [inline Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1:752]: Again, need a VSCode root repository search on this.
Are you sure you're covering all variants of AAD?
1. AAD
2. Azure Active Directory

**Review by aakashpahwa-msft — CHANGES_REQUESTED** [review]

File: Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1 (hunk 2 above)
Comment (rajatgoyalMS) [reply Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1]: fixed.

File: Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1 (hunk 1 above)
Comment (rajatgoyalMS) [reply Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1]: for control Ids - Mircosoft_Entra_ID is being used,
all other places Entra Id is used.

File: Control coverage/Feature/APIManagement.md [hunk 4]
```diff
@@ -11,7 +11,7 @@
 - [Azure_APIManagement_AuthZ_Validate_JWT](#azure_apimanagement_authz_validate_jwt)
 - [Azure_APIManagement_AuthN_Disable_Management_API](#azure_apimanagement_authn_disable_management_api)
 - [Azure_APIManagement_AuthZ_Enable_User_Authorization_For_API](#azure_apimanagement_authz_enable_user_authorization_for_api)
-- [Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN](#azure_apimanagement_authn_use_aad_for_client_authn)
+- [Azure_APIManagement_AuthN_Use_Microsoft_Entra_ID_for_Client_AuthN](#azure_apimanagement_authn_use_microsoft_entra_id_for_client_authn)
```
Comment (Aboli-msft) [inline Control coverage/Feature/APIManagement.md:14]: update later reference for this control in same ffile

Comment (rajatgoyalMS) [reply Control coverage/Feature/APIManagement.md:14]: done.

**Review by Aboli-msft — APPROVED** [review]

**Review by aakashpahwa-msft — APPROVED** [review]

**Pull request merged**